packages/vue/index.js

```diff
@@ -0,0 +1,57 @@
+const web = require('@neutrinojs/web');
+const path = require('path');
+const merge = require('deepmerge');
+const loaderMerge = require('@neutrinojs/loader-merge');
+
+const MODULES = path.join(__dirname, 'node_modules');
+
+module.exports = (neutrino, options) => {
+  neutrino.use(web, options);
+  neutrino.options.extensions = ['vue']; // eslint-disable-line no-param-reassign
```

Comment:
We now have `vue` in the extensions list, so we don't need this line now.

Reply:
If I include the `js` extension, I get an error:

```
ERROR in   Error: The loader "/Users/haali/Documents/Mozilla/projects/neutrino-dev/node_modules/html-webpack-plugin/lib/loader.js!/Use  rs/haali/Documents/Mozilla/projects/neutrino-dev/node_modules/html-webpack-template/index.ejs" didn't return html.
```

If extensions is something like `['jsx', 'vue', 'ts', 'mjs' ]` (without the js extension), then it works. Do you know why this is happening?

Reply:
Also, should we include `.json` in the list of extensions? :)

Reply:
Looks like this:

https://github.com/jantimon/html-webpack-plugin/issues/602

Basically, I think the vue-loader is only supposed to accept `vue` extensions:

```js
neutrino.config.module
    .rule('vue')
      .test(/\.vue$/) // instead of .test(neutrino.regexFromExtensions())
      .use('vue')
        .loader(require.resolve('vue-loader'))
        .options(options);
```

Reply:
> Also, should we include .json in the list of extensions? :)

I...don't know. Give it a shot and see if the tests continue to pass, but I'm not sure what effect it may have.

Reply:
That's what I figured, sounds good!

Btw, instead of hardcoding `/\.vue$/`, we could just do `neutrino.regexFromExtensions(['vue'])`.

Reply:
:heart_eyes: